ansible/roles/fluentd-node-collector/vars/main.yaml

```diff
@@ -1,5 +1,6 @@
 ---
-ansible_ssh_user: root
+#ansible_ssh_user: cloud-user
```

Comment:
removed this, I think it's better to have this var in inventory, but that depends on a particular setup you run against.